Assets/MRTK/Core/Providers/BaseController.cs

```diff
@@ -242,14 +242,8 @@ protected virtual bool TryRenderControllerModel(Type controllerType, InputSource
                 return true;
             }
 
-            GameObject controllerModel = null;
-            bool usePlatformModels = controllerVisualizationProfile.GetUsePlatformModelsOverride(controllerType, ControllerHandedness);
```

Comment:
I'm slightly confused on this one. Was the fallback not actually working previously? It seems like we'd want to attempt the platform models before loading the override.

Reply:
Yeah it wasn't working previously.

The outer method actually tries to load the platform-provided first, this check erroneously loads the global default model instead of the platform override specified model.